test/pylib/manager_client.py

```diff
@@ -135,10 +136,17 @@ async def server_add(self) -> str:
         logger.debug("ManagerClient added %s", server_id)
         return server_id
 
-    async def server_remove(self, server_id: str) -> None:
+    async def server_remove(self, to_remove_ip: str) -> None:
         """Remove a specified server"""
-        logger.debug("ManagerClient removing %s", server_id)
-        await self.cli.get_text(f"/cluster/removeserver/{server_id}")
+        logger.debug("ManagerClient removing server %s", to_remove_ip)
+        await self.cli.get_text(f"/cluster/remove-server/{to_remove_ip}")
+        self._driver_update()
+
+    # TODO: only pass UUID
+    async def remove_node(self, to_remove_ip: str, to_remove_uuid: str) -> None:
```

Comment:
The IP given should be the IP of the server who does the remove.

Reply:
also rename `to_remove_uuid` to `to_remove_host_id`.

Reply:
Done in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d